completed-TS-number 9

numerouno-life opened the pull request.

numerouno-life committed:
completed-TS-number 9

numerouno-life committed:
completed-TS-number 9

numerouno-life committed:
completed-TS-number 9

irek-valiev commented on src/tracker/http/handler/EpicHandler.java:
```diff
@@ -0,0 +1,92 @@
+        try {
+            if (id == null) {
+                taskManager.createEpic(epic);
+                sendText(exchange, "", 201);
```
поскольку теперь речь идет не о локальном использовании библиотеки, а через какую-то сеть, надо пользователю сообщить результаты работы над сущностью, т.е. в ответе передать еще id созданного эпика

irek-valiev commented on src/tracker/http/handler/SubtaskHandler.java:
```diff
@@ -0,0 +1,94 @@
+        try {
+            if (id == null) {
+                taskManager.createSubtask(subtask);
+                sendText(exchange, "", 201);
```
аналогично для подзадачи

irek-valiev commented on src/tracker/http/handler/TaskHandler.java:
```diff
@@ -0,0 +1,92 @@
+        try {
+            if (id == null) {
+                taskManager.createTask(task);
+                sendText(exchange, "", 201);
```
аналогично для задачи

numerouno-life committed:
completed-TS-number 9

numerouno-life committed:
reworked-TS-number 9
Добавил сообщения для пользователя при "Создании", "Обновлении" и "Удалении" всех видов задач
